### DISCUSSION TITLE
Can this software use GPU instead of CPU?

### DISCUSSION THREAD

**Discussion by drvectorio** [yermak/AudioBookConverter#613] [Q&A]
It would be nice to convert files with GPU (nvidia), is there any possibility to do this with GPU?

**Comment by yermak** [comment]
That would be my dream, however, I don't think there are existing audio encoders based on GPU, unlike video audio considering to be much easier to encode - I don't think anyone is going to invest into this with such limited benefits.
In theory it is doable (especially as a proof of concept), but it's a big project to make it production ready.
